admin/authentication/local-strategy.js

```diff
@@ -28,7 +28,7 @@ module.exports = async function (req, email, password, done) {
     // school with dfeNumber 1234567.
     // NOTE that this is not a generic implementation: you actually need to be called 'helpdesk' or 'service-manager'
     // rather than simply have the role.  This is fine - it's only for local-dev
-    const matches = email.match(/^(helpdesk|service-manager):(\d{7})$/)
+    const matches = email.match(/^(service-manager):(\d{7})$/)
```

Comment:
@GuyHarwood @jon-shipley I removed the feature to directly impersonate a school from login page as this really serves no purpose now that the helpdesk site exists. Happy to discuss this further if needed.

Reply:
makes sense